main.go

```diff
@@ -547,14 +555,15 @@ func main() {
 		os.Exit(1)
 	}
 	if err = (&controllers.HCPVaultSecretsAppReconciler{
-		Client:                      mgr.GetClient(),
-		Scheme:                      mgr.GetScheme(),
-		Recorder:                    mgr.GetEventRecorderFor("HCPVaultSecretsApp"),
-		SecretDataBuilder:           secretDataBuilder,
-		HMACValidator:               hmacValidator,
-		MinRefreshAfter:             minRefreshAfterHVSA,
-		BackOffRegistry:             controllers.NewBackOffRegistry(backoffOpts...),
-		GlobalTransformationOptions: globalTransOptions,
+		Client:                              mgr.GetClient(),
+		Scheme:                              mgr.GetScheme(),
+		Recorder:                            mgr.GetEventRecorderFor("HCPVaultSecretsApp"),
+		SecretDataBuilder:                   secretDataBuilder,
+		HMACValidator:                       hmacValidator,
+		MinRefreshAfter:                     minRefreshAfterHVSA,
+		BackOffRegistry:                     controllers.NewBackOffRegistry(backoffOpts...),
+		GlobalTransformationOptions:         globalTransOptions,
+		CleanupOrphanedShadowSecretInterval: cleanupOrphanedShadowSecretInterval,
```

Comment:
might be good to pass this to SetupWithManager() instead of extending the set of exported members.